typed_python/_runtime.cpp

```diff
@@ -299,20 +300,139 @@ extern "C" {
         return floorresult;
     }
 
-    PyObject* nativepython_runtime_int_to_pyobj(int64_t i) {
+    // returns layout*
+    void *np_runtime_pyobj_to_typed(PyObject *obj, void* tgt, uint64_t tp) {
```

Comment:
tp should be a Type*, and tgt an instance_ptr. It will do the same thing (llvm will pass everything to us in 64 bit registers) but it makes the code clearer.

Reply:
Regarding tgt: tgt is a pointer to a layout, and &tgt is an instance_ptr. So I wasn't sure what type to assign to tgt to represent pointer to a layout.

Reply:
These changes have been made, this branch is superseded by jeff-conv4, pull request #208.